osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/map.js

```diff
@@ -74,7 +84,7 @@
 
         this._setLocationFilterFromExcerptID = function(ID) {
             var that = this;
-            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/").on('data:loaded', function(){
+            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/?format=json").on('data:loaded', function(){
```

Comment:
It was not working without specifying `format=json` so I hat to use that. I'm open for a other solution...